docs/integrations/optuna.rst

```diff
@@ -3,150 +3,214 @@
 
-Pass **neptune_monitor** to **study.optimize**
-----------------------------------------------
-Monitor the metrics and parameters checked at each run:
+* ``neptune-client==0.4.125`` or newer, and ``neptune-contrib==0.24.7`` or newer: See :ref:`neptune-client installation guide <installation-neptune-client>`.
+* ``optuna==2.3.0``. See |optuna-install|.
 
-.. code-block:: python3
+You also need minimal familiarity with Optuna. Have a look at the |optuna-guide| guide to get started.
 
-    study = optuna.create_study(direction='maximize')
-    study.optimize(objective, n_trials=100, callbacks=[neptune_callback])
+.. code-block:: bash
+	
+   pip install --quiet optuna neptune-client neptune-contrib['monitoring']
 
-Log all results
----------------
-You can log additional information from optuna study after the sweep has completed.
-By running:
+Step 1: Initialize Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Run the code below:
 
 .. code-block:: python3
 
-    opt_utils.log_study_info(study)
+    import neptune
 
-You log the following things to Neptune:
+    neptune.init(api_token='ANONYMOUS', project_qualified_name='shared/showroom')
 
-* Best score
-* Best parameters
-* Interactive plotly figures from optuna.visualization: plot_contour, plot_slice, plot_parallel_coordinate, optimization_history
-* Pickled study object
+.. tip::
 
-.. image:: ../_static/images/integrations/optuna_charts.gif
-   :target: ../_static/images/integrations/optuna_charts.gif
-   :alt: Optuna charts in Neptune
+    You can also use your personal API token. Read more about how to :ref:`securely set the Neptune API token <how-to-setup-api-token>`.
 
-Log study and charts after every iteration
-------------------------------------------
-You can log interactive charts from optuna.visualization and the study object after every iteration.
-Simply specify what you want to log at `NeptuneCallback` creation.
+Step 2: Create an Experiment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Run the code below to create a Neptune experiment:
 
 .. code-block:: python3
 
-    neptune_callback = opt_utils.NeptuneCallback(log_study=True, log_charts=True)
+    neptune.create_experiment('optuna-sweep')
 
+Open the link in a new tab. The charts will currently be empty, but keep the window open. You will be able to see live metrics once logging starts.
 
-Monitor your Optuna training in Neptune
----------------------------------------
-Now you can watch your Optuna hyperparameter optimization in Neptune!
+Step 3: Create the Neptune Callback
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. code-block:: python3
 
-Check out this |example experiment|.
+   import neptunecontrib.monitoring.optuna as opt_utils
 
-.. image:: ../_static/images/integrations/optuna_monitoring.gif
-   :target: ../_static/images/integrations/optuna_monitoring.gif
-   :alt: Optuna monitoring in Neptune
+   neptune_callback = opt_utils.NeptuneCallback()
 
-Full script
------------
+Step 4: Running Optuna with the Neptune callback
```

Comment:
You were using the tone "create..." so perhaps it should be "Run Optuna study with the NeptuneCallback"